crates/astria-sequencer/src/transaction/checks.rs

```diff
@@ -100,6 +100,7 @@ pub(crate) async fn get_fees_for_transaction<S: StateRead>(
         .context("failed to get bridge sudo change fee")?;
 
     let mut fees_by_asset = HashMap::new();
+    let mut tx_index_of_action = 0;
     for action in &tx.actions {
```

Comment:
Don't manually increment a counter. Use `tx.actions.iter().enumerate()` instead.

Reply:
Recommenting: the intention was to do the below instead of manually driving the iterator
```suggestion
    for (i, act) in tx.actions().iter().enumerate() {
```